static/js/stream_create.js

```diff
@@ -150,18 +150,28 @@ function create_stream() {
 
     let invite_only;
     let history_public_to_subscribers;
+    let is_web_public;
     const privacy_setting = $('#stream_creation_form input[name=privacy]:checked').val();
 
     if (privacy_setting === 'invite-only') {
+        is_web_public = false;
```

Comment:
This is probably fine for now, but we might want a follow up commit later that only sends fields that are really necessary.  I think we can say `invite_only == True` implies `is_web_public == False` on the server.

And then later:

~~~
if (is_web_public !== undefined) {
   data.is_web_public = JSON.stringify(is_web_public):
}
if (invite_only !== undefined) {
   // etc.